app/Http/Controllers/RegisterController.php

```diff
@@ -33,7 +33,7 @@ public function registerUser(Request $request,Accountant $accountant){
         $Accountant->password = Hash::make($request->password);
         $res= $Accountant-> save();
         if ($res){
-            return back()->with('success','You have Registered Successfully');
+            return redirect('login');
```

Comment:
```suggestion
            return redirect()->route('login');
```
Instead of using direct urls for routes, recommended is to define the route name and use Laravel redirect to named route. So, in case of any url updates, less code refactors needed to be performed.

Reply:
Similar changes on other places also.

Reply:
PR updated as per reviews.